edgelet/build/windows/util.ps1

```diff
@@ -1,57 +1,216 @@
-# Copyright (c) Microsoft. All rights reserved.
-
-New-Item -Type Directory -Force '~/.cargo/bin'
-$env:PATH += ";$(Resolve-Path '~/.cargo/bin')"
-
-function Test-RustUp
-{
-    (get-command -Name rustup.exe -ErrorAction SilentlyContinue) -ne $null
-}
-
-function Get-CargoCommand
-{
-    if (Test-RustUp)
-    {
-        'cargo +stable-x86_64-pc-windows-msvc '
-    }
-    else
-    {
-        "$env:USERPROFILE/.cargo/bin/cargo.exe +stable-x86_64-pc-windows-msvc "
-    }
-}
-
-function Get-Manifest
-{
-    $ProjectRoot = Join-Path -Path $PSScriptRoot -ChildPath "../../.."
-    Join-Path -Path $ProjectRoot -ChildPath "edgelet/Cargo.toml"
-}
-
-function Assert-Rust
-{
-    Write-Host "Validating Rust (stable-x86_64-pc-windows-msvc) is installed and up to date."
-    if (-not (Test-RustUp))
-    {
-        Write-Host "Installing rustup and stable-x86_64-pc-windows-msvc Rust."
-        Invoke-RestMethod -usebasicparsing 'https://static.rust-lang.org/rustup/dist/i686-pc-windows-gnu/rustup-init.exe' -outfile 'rustup-init.exe'
-        if ($LastExitCode)
-        {
-            Throw "Failed to download rustup with exit code $LastExitCode"
-        }
-
-        Write-Host "Running rustup-init.exe"
-        ./rustup-init.exe -y --default-toolchain stable-x86_64-pc-windows-msvc
-        if ($LastExitCode)
-        {
-            Throw "Failed to install rust with exit code $LastExitCode"
-        }
-    }
-    else
-    {
-        Write-Host "Running rustup.exe"
-        rustup install stable-x86_64-pc-windows-msvc
-        if ($LastExitCode)
-        {
-            Throw "Failed to install rust with exit code $LastExitCode"
-        }
-    }
-}
+# Copyright (c) Microsoft. All rights reserved.

```

Comment:
Seems you changed the line endings of this file. Revert it to whatever it was before.